test/e2e/provisioning/helper.go

```diff
@@ -110,10 +110,13 @@ func testScenario(t *testing.T, testCase scenario, cloudProvider string, testPar
 	// only used by OpenStack scenarios
 	scenarioParams = append(scenarioParams, fmt.Sprintf("<< OS_IMAGE >>=%s", openStackImages[testCase.osName]))
 
-	gopath := os.Getenv("GOPATH")
```

Comment:
This must be the other way around, first check the kubeconfig in the repo path, otherwise you might break the existing behavior.

Also please update the Readme

Reply:
Done.